seed-job/template.yaml

```diff
@@ -164,7 +169,7 @@ objects:
                         // we need to make sure the image is removed in either case
                         stage('Remove the image') {
                           // don't fail the pipeline if image removal failed due to some error
-                          sh (script: "docker rmi ${image_name_with_registry}; docker rmi ${image_name}", returnStatus: true)
+                          sh (script: "buildah rmi ${image_name_with_registry}; buildah rmi ${image_name_with_registry}", returnStatus: true)
                           def output = sh(returnStdout:true, script: "docker rmi ${base_image_name} > image-remove 2>&1", returnStatus: true)
                           sh "cat image-remove"
                           sh(script: "docker rmi `docker images -q -f dangling=true`", returnStatus: true)
```

Comment:
@dharmit : do we need to also make sure we remove the dangling images (if any at all) managed by buildah ?

Reply:
Yes, we will have to take that into account. I haven't considered that in this PR yet. Thanks for reminding. :smile: